src/main/java/se/sundsvall/casemanagement/integration/ecos/PartyService.java

```diff
@@ -162,9 +162,10 @@ private ArrayOfguid getEcosFacilityRoles(final StakeholderDTO s) {
 			.withGuid(s.getRoles().stream()
 				.map(roleString ->
 				{
-					final var r	= StakeholderRole.valueOf(roleString);
+					final var r = StakeholderRole.valueOf(roleString);
```

Comment:
final var role perhaps? (and the parameter could be something cooler than s perhaps :D